### DISCUSSION TITLE
error writing JSON status 204

### DISCUSSION THREAD

**Discussion by aoberest** [seaweedfs/seaweedfs#2155] [Q&A] [answered]
Hello.

What is mean this log message ? 
```
I0625 09:17:01 44757 common.go:86] error writing JSON status 204: http: request method or response status code does not allow body
```

**Comment by aoberest** [comment, marked as answer]
I finded this issiue https://github.com/chrislusf/seaweedfs/issues/1196
but i never receive it earlier.
How to reaction with the message ?

**Reply by chrislusf** [reply]
I can not tell when it would happen.

**Reply by aoberest** [reply]
Today I connected the  [Gitlab Consolidated object storage](https://docs.gitlab.com/ee/administration/object_storage.html#consolidated-object-storage-configuration) to the s3 seaweedfs cluster and received this message in the log.

Could you please explain to me why I am receiving these messages?
Should I be worried about this?

**Reply by chrislusf** [reply]
you can skip that.

**Reply by aoberest** [reply]
Understood. Thanks for the reply.
